File: curso_c_cpp/c/05_condicionales/ejemplos/calificacion.c
/*
 * else if encadenado: convertir una nota numerica a una letra.
 */
#include <stdio.h>

int main(void) {
    int nota;
    printf("Nota (0-100): ");
    scanf("%d", &nota);

    if (nota >= 90) {
        printf("A - Excelente\n");
    } else if (nota >= 80) {
        printf("B - Muy bien\n");
    } else if (nota >= 70) {
        printf("C - Aprobado\n");
    } else if (nota >= 60) {
        printf("D - Justo\n");
    } else {
        printf("F - Reprobado\n");
    }
    return 0;
}

File: curso_java/05_condicionales/ejemplos/Calificacion.java
/*
 * else if encadenado: nota numerica a letra.
 */
import java.util.Scanner;

public class Calificacion {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Nota (0-100): ");
        int nota = sc.nextInt();

        if (nota >= 90) {
            System.out.println("A - Excelente");
        } else if (nota >= 80) {
            System.out.println("B - Muy bien");
        } else if (nota >= 70) {
            System.out.println("C - Aprobado");
        } else if (nota >= 60) {
            System.out.println("D - Justo");
        } else {
            System.out.println("F - Reprobado");
        }
        sc.close();
    }
}
